config/base/kustomization.yaml

```diff
@@ -14,3 +12,5 @@ images:
   newTag: v1
 resources:
 - ../manager
+patchesStrategicMerge:
+- manager_auth_proxy_patch.yaml
```

Comment:
why the change

Reply:
This actually comes from @akartsky 's latest changes. I switched the order of his `make` targets so that it runs the US target last, that way all of our Kustomize templates have the US implementation as the default when checked in.

Reply:
wow, good thinking.